### PR TITLE
Add MPS backend for ADD operation

### PR DESCRIPTION
**Pull request by sunlin-link**

### CODE REVIEW

**Comment by sunlin-link** [comment]
Fixes:
https://github.com/intel/webml-polyfill/issues/92

**Comment by ibelem** [comment]
@huningxin please help to review this PR.

File: services/ml/execution_impl_mac.mm [hunk 1]
```diff
@@ -512,6 +514,10 @@ bool GetMPSImageInfo(const OperandMac& operand,
                                               .threadsPerThreadgroup];
                 [encoder endEncoding];
               }
+            } else if (binary_kernel) {
+              [binary_kernel encodeToCommandBuffer:command_buffer
+                             primaryImage:src_img
+                             secondaryImage:dst_img];
```
Comment (huningxin) [inline services/ml/execution_impl_mac.mm]: This seems not right to me. I think this should call [`encodeToCommandBuffer:primaryImage:secondaryImage:destinationImage:`](https://developer.apple.com/documentation/metalperformanceshaders/mpscnnbinarykernel/2865636-encodetocommandbuffer?language=objc). I don't understand the reason to use `dst_img` to `secondaryImage`. I think you should set the `dst_img` for `destinationImage`. And you need to set right parameter to `secondaryImage`.

**Comment by huningxin** [comment]
I wondering which test cases you used to verify your implementation. 

As I know, there are at least two test cases for ADD op:
https://github.com/intel/webml-polyfill/blob/master/test/operations/add.js
https://github.com/intel/webml-polyfill/blob/master/test/cts/test/V1_0/add.js

However, in my test, they all failed on this PR.

@BruceDai , please add any test cases for ADD if I missed.

**Comment by BruceDai** [comment]
@huningxin Yes, currently there are above two test cases for ADD op, more tests for ADD ops with different Fused activation function types(FUSED_RELU/FUSED_RELU1/FUSED_RELU6) will add soon.

**Comment by huningxin** [comment]
https://github.com/otcshare/chromium-src/pull/36/commits/fc5b1fc4d787a4caf798365a2f7c65ddc9a32637 has been included in https://github.com/otcshare/chromium-src/pull/40 which completes the `ADD` (and `MUL`) support. So close this one.

**Pull request closed without merging**